edm4hep.yaml

```diff
@@ -489,3 +517,74 @@ datatypes :
     OneToOneRelations:
      - edm4hep::ReconstructedParticle rec // reference to the reconstructed particle
      - edm4hep::Vertex vertex             // reference to the vertex
+
+
+  #----------- SimPrimaryIonizationCluster
+  edm4hep::SimPrimaryIonizationCluster:
+    Description: "Simulated Primary Ionization"
+    Author : "Wenxing Fang, IHEP"
+    Members:
+      - uint64_t cellID                      //cell id.
+      - float time                           //the primary ionization's time in the lab frame in [ns].
+      - edm4hep::Vector3d position           //the primary ionization's position in [mm].
+      - int16_t type                         //type.
```

Comment:
You change this from `int` to `int16_t`, however, `int` is usually 32 bits on 64 bit systems in c++. I am fine with either, just want to make sure that 16 bits is enough here.